File: DSA/upama_kumari_590014909/day_16_question_2.py
def count_frequency(arr, target):
    count = 0
    for num in arr:
        if num == target:
            count += 1
    return count

array = [5, 1, 5, 5, 3, 5, 7]
target = 5
result = count_frequency(array, target)
print(result)

File: DSA/ShatakshiMishra_590015008/Day16Question2.cpp
#include <iostream>
#include <vector>
using namespace std;

int countOccurrences(vector<int>& arr, int target) {
    int count = 0;
    for (int num : arr) {
        if (num == target)
            count++;
    }
    return count;
}

int main() {
    int n;
    cin >> n;
    vector<int> arr(n);
    for(int i=0;i<n;i++)
    {
        cin >> arr[i];
    }
    int target;
    cin >> target;

    int result = countOccurrences(arr, target);

    cout << result << endl;
}
